README.md

```diff
@@ -19,7 +18,52 @@ In summary:
   This is a single extract for variables that are not expected to change, such as ethnicity.
 * Use cohort-joiner to join the cohorts.
 
-[_project.yaml_]: https://github.com/opensafely-actions/cohort-joiner
+Let's walk through an example _project.yaml_.
+
+The following cohort-extractor action extracts the left-hand cohorts:
+
+```yaml
+generate_cohort:
+  run: >
+    cohortextractor:latest generate_cohort
+      --study-definition study_definition
+      --index-date-range "2021-01-01 to 2021-06-30 by month"
+  outputs:
+    highly_sensitive:
+      cohort: output/input_2021-*.csv
+```
+
+The following cohort-extractor action extracts the right hand cohort:
+
+```yaml
+generate_ethnicity_cohort:
+  run: >
+    cohortextractor:latest generate_cohort
+      --study-definition study_definition_ethnicity
+  outputs:
+    highly_sensitive:
+      cohort: output/input_ethnicity.csv
+```
+
+Finally, the following cohort-joiner reusable action joins the cohorts.
+Remember to replace `[version]` with [a cohort-joiner version][1]:
+
+```yaml
+join_cohorts:
+  run: >
+    cohort-joiner:[version]
+      --lhs output/input_2021-*.csv
```

Comment:
I think we never explain what `lhs` stands for and in the text we write "left hand cohort". I think left hand cohort is clearer but still wondered if we should explain this once somewhere for people who are not familiar with these abbreviations and joins? or did I miss where we explain this, I think @LFISHER7 mentioned something similar in a previous review?